src/main/java/duke/Task.java

```diff
@@ -0,0 +1,68 @@
+package duke;
+
+/**
+ * Class to handle the task
+ */
+public class Task {

```

Comment:
i like that all task type classes are inherited from a parent class task, which prevents repeating many lines of code